comp/logs/agent/flare/flare_controller_test.go

```diff
@@ -0,0 +1,50 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package flare
+
+import (
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+
+	"github.com/DataDog/datadog-agent/comp/core/flare/helpers"
+)
+
+type FlareControllerTestSuite struct {
```

Comment:
I don't think `FlareControllerTestSuite` is used anywhere? if so - you can remove this.

Reply:
Removed